docs/topics/proc_deploying-the-rest-api-booster-using-the-oc-cli-client.adoc

```diff
@@ -9,11 +9,15 @@
 
 .Prerequisites
 
-* The booster application created using the {launcher} application.
+ifndef::parameter-openshiftlocal[]
```

Comment:
Same as `proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc`.